creator-node/src/services/stateMachineManager/clockUtils.js

```diff
@@ -0,0 +1,130 @@
+const { CreatorNode } = require('@audius/libs')
+const axios = require('axios')
+const retry = require('async-retry')
+
+const config = require('../../config')
+const { logger } = require('../../logging')
+const { generateTimestampAndSignature } = require('../../apiSigning')
+const {
+  BATCH_CLOCK_STATUS_REQUEST_TIMEOUT,
+  CLOCK_STATUS_REQUEST_TIMEOUT_MS,
+  MAX_USER_BATCH_CLOCK_FETCH_RETRIES
+} = require('./constants')
+
+const MAX_BATCH_CLOCK_STATUS_BATCH_SIZE = config.get(
+  'maxBatchClockStatusBatchSize'
+)
+const SP_ID = config.get('spID')
+const DELEGATE_PRIVATE_KEY = config.get('delegatePrivateKey')
+
+class ClockUtils {
```

Comment:
this doesn't correspond 1-1 with any snapback module. it just extracts a couple of clock-related functions that were in `snapbackSM.js`

Reply:
given this is basically just one function, i'd be down to just make that the main export from here + rename file to mirror func name

maybe we can create a `stateMachineManager/utils` and throw this in there to make the root dir pretty

Reply:
I don't think it's just one function tho because `_retrieveClockValueForUserFromReplica` is used elsewhere independently of `retrieveClockStatusesForUsersAcrossReplicaSet`. I'd be happy to move both clockUtils.js and utils.js to a utils folder for cleanliness 👍

Reply:
synced offline. should remove the underscore prefix since it's no longer private to the snapbackSM and maybe export individual functions instead of class (nbd)

Reply:
also consolidate clockUtils into utils if it's not going to grow

Reply:
done ✅